virttest/qemu_devices/qcontainer.py

```diff
@@ -942,6 +942,27 @@ def get_aavmf_vars(params):
                                                   parent_bus={'aobject': 'pci.0'}))
             return devices
 
+        def machine_s390x_virtio(cmd=False):
+            """
+            s390x (s390) doesn't support PCI bus.
```

Comment:
nitpick: qemu calls this machine `s390-ccw-virtio` and the avocado-vt machine profile is called `s390-virtio`. I think it'd be best to keep the names in sync and use `machine_s390_virtio`